src/app/services/user/user.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection, AngularFirestoreDocument, AngularFirestoreModule } from 'angularfire2/firestore';
+import { Observable } from 'rxjs/Observable';
+import { User } from '../../models/User';
+
+@Injectable()
+export class UserService {
```

Comment:
It would be cool to have method stubs, returning some mock data: 
 - getCurrentUserName():String;
- isUserAuthorized(): boolean;

These methods're gonna be used in #7 _Add pages header_ issue.

Reply:
CRUD operations is not allowed for users. We're using Firebase Auth, so,seems, it's not necessary.
What do u think?

I think we're not gonna use any operation with users, maybe let's mark the methods as deprecated.